src/session.js

```diff
@@ -30,6 +26,11 @@ export default class Session extends EventEmitter {
         };
     }
 
+    static _generateSessionId () {
+        // NOTE: #116 (https://github.com/superroma/testcafe-hammerhead/issues/116)
+        return uuid.v4().substr(0, 3);
```

Comment:
@churkin forgot to mention: do not use full link in comments, because we will move from the superroma account.
Use `testcafe-hammerhead#16` format instead